src/js/config/runtimeConfig.template.js

```diff
@@ -0,0 +1,12 @@
+var runtimeConfig = {
+  // The domain used to compute the service endpoint
+  serviceDomain: "marathon-par.central.criteo.preprod",
+
+  // The pattern to build the links to the logs
+  appLogsLinkPattern: "http://kibana.par.preprod.crto.in/app/kibana#/discover?\
```

Comment:
criteo specific urls. Do you need to have these in open source code?

Reply:
And furthermore those URLs do work only for PAR and prod

Reply:
@kamaradclimber , you are right, I will replace it with something unrelated like http://mylogs/?appId={{appId}}. This is just a template.

@pierresouchay, this "pattern" (that I will rename "template" btw) is supposed to be different in each DC and env. This is why I leveraged the concept of runtimeConfig brought by Cedric. That way, we won't have to rebuild the package for every (dc, env).